core/src/Kokkos_CudaSpace.hpp

```diff
@@ -57,6 +57,23 @@
 
 #include <Cuda/Kokkos_Cuda_abort.hpp>
 
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_PIN_UM_TO_HOST
+#ifndef KOKKOS_IMPL_DEFINED_CUDA_PIN_UM_TO_HOST_V
+extern bool kokkos_impl_cuda_pin_um_to_host_v;
```

Comment:
Any reason why this cannot be a static private data member of `CudaUVMSpace` rather the a global variable?

Reply:
This was a way which worked for us in conjunction with writing an external tool to set the variable (i.e. like in the profiling tools). I guess we could make the tool fully dependent on Kokkos and/or check whether declaring the setter/getter functions extern in the tool would work too.